src/test/java/au/org/democracydevelopers/raireservice/service/GenerateAssertionsOnKnownTestCases.java

```diff
@@ -21,10 +21,10 @@ the raire assertion generation engine (https://github.com/DemocracyDevelopers/ra
 package au.org.democracydevelopers.raireservice.service;
 
 
```

Comment:
Some of the lines of code in this file are too long.

Reply:
Cut to 110.